config/alpha.json

```diff
@@ -27,7 +27,7 @@
     "appId": "saturnnonprod-O9lUP",
     "apiKey": "AQEBBAEkx4iE2KxNyI7Wx08EwU1ycTM7E4FMSmaibbMUQxNU6uQvuAJt7fyABAtFYSYfgEE"
   },
-  "azurePreviewGroup": "alpha-azure-feature-group",
+  "azurePreviewGroup": true,
```

Comment:
I decided that removing this key entirely from non-prod config files would be more confusing than leaving it but making it clear that we always use true for those environments. The only somewhat ugly part of this choice is that the value is now either a boolean or a Sam group name.